tests/roundtrip_otf.py

```diff
@@ -0,0 +1,34 @@
+import extractor
```

Comment:
this is not really a test

but maybe it should become one, different PR 
I would consider Plex as examples and have smaller scale examples of otf, ttf, pfa, pfb, woff, woff2 to extract and compare it with provided ufo data.

Reply:
Yeah, I know it's not a proper test. More a script to make sure everything works. I actually began writing proper tests, but as you say, they should better be in a different PR.